README.md

```diff
@@ -57,3 +57,27 @@ in ``com.google.u2f.tools.httpserver.UtfHttpServer.java`` and runs on port
 8080. Run this class as a regular Java application (right click, select *Run
 As* and *Java Application*). Note that you need to have the U2F extension
 installed in Chrome in order for the demo app to talk to your U2F token.
+
+### U2F-GAE-Demo
+
+The u2f-gae-demo project is a sample application built on the Google Appengine web platform which
+demonstrates a possible UX for user interaction with U2F in a web page.
+
+As above, after importing the project into Eclipse you might have to adjust JDK 
+versions,
+AppEngine SDK version, etc. Once everything compiles, you can run the Appengine server locally and point 
+Google Chrome at http://localhost:8888/. The built-in support for U2F in Google Chrome only works on HTTPS sites.
+To test the app on (the non-https origin) http://localhost:8888, you need to do the following:
+
+* Quit Google Chrome.
+* Re-start Google Chrome with the `--show-component-extension-options` command-line flag.
```

Comment:
I don't agree that restarting Chrome with a command-line flag is easier than loading an extension from the web store, especially since fully killing Chrome can be troublesome, depending on the platform. I won't argue strenuously, but if you haven't lately, would you mind trying the install-an-extension workflow to confirm your opinion?